pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Note - this is fine, I'll just have to update my pnpm locally to the latest

Reply:
Great. I have the corepack enabled & it takes the pnpm version from this repo's package.json & still gave me this ouput. So, I thought this should be fine.